src/html.coffee

```diff
@@ -1194,7 +1202,8 @@ do ->
   htmlModule.table.body = ({parent, data, compare, rowClass, classStr, rowLimit, columns, on:onRow}) ->
     throw new Error('there needs to be a parent') unless parent
     throw new Error('there needs to be data') unless data
-    tbody = dvl.valueOf(parent).append('tbody').attr('class', classStr)
+    parent = dvl.valueOf(parent)
+    tbody = parent.append('tbody').merge(parent).attr('class', classStr)
```

Comment:
it doesn't look like it's possible to leave a comment outside of any changes on github, but at lines 1374-1376, I noticed those methods might need to be flattened from `d3.scale.linear` -> `d3.scaleLinear` and `d3.svg.line` -> `d3.line` (https://github.com/d3/d3/blob/master/CHANGES.md#shapes-d3-shape)

Reply:
good catch... yeah, they do need to be flattened.  The tests must not be exercising that code... I'll see if I can add one that does.

Reply:
ugh, none of the methods in the `htmlModule` are tested.  I definitely don't think it's worth the time it will take to set up a test suite for this -- in general DVL is obsolete and we need to migrate off it anyway, and specifically we aren't using this `sparkline` method anywhere.